### ISSUE TITLE
Needs NMake/MSVS Support

### ISSUE THREAD

**Issue by tilkinsc** [tilkinsc/LuaConsole#10]
* A project file from different versions
* NMake files that MSVS supports

**Comment by tilkinsc** [comment]
Problems:
* Its not built for MSVS libs
* Its not built for directory-less include files (package your libs with the source code)
* Includes things not found in MSVS (dirent for one)
* Might not have a few other symbols found relating to ^
* No definition for MSVS under _WIN32 || _WIN64 ifdefs
* luaadditions not built correctly or even a little bit

**Comment by tilkinsc** [comment]
So I did more research and attempted to build lua and LuaConsole via CL.exe and I got it working!

So it was pretty gross, but I managed to gather all files together and do it in two different places (lua-5.3.5 folder for lua, and src folder for LuaConsole). All things aside I can now specify relative paths and migrate things around as I do in the build.bat for mingw.

Everything compiles good, but I got an error on a falsely static on the HELP_MESSAGE variable in ldata.h so I will be fixing that up. There is now no more need for detecting MSVS, as it should be a separate build file named accordingly.
